include/nighthawk/distributor/nighthawk_distributor_client.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "envoy/common/pure.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/distributor/distributor.grpc.pb.h"
+
+namespace Nighthawk {
+
+/**
+ * Interface of a gRPC distributor service client.
+ */
+class NighthawkDistributorClient {
+public:
+  virtual ~NighthawkDistributorClient() = default;
+
+  /**
+   * @brief Propagate messages to one or more other services for handling.
```

Comment:
nit: I don't think we need brief here, because this is already only one line.

Reply:
Fixed in 4185e3ce